projects/management-software-sync/backend/samples/schema/products.json

```diff
@@ -1,15 +1,15 @@
 [
-    {
-        "id": "",
-        "id_catalogo": "",
-        "new": "",
-        "oe-am": "",
-        "descrizione": "",
-        "sottogruppo": "",
-        "listino": "",
-        "quantita_minima": "",
-        "unita_misura": "",
-        "pezzi": "",
-        "confezione": ""
-    }
-]
\ No newline at end of file
+  {
+    "id": "",
+    "id_catalogo": "",
+    "new": "",
+    "oe-am": "",
```

Comment:
Not clear too.

Reply:
"oe-am" can't be translated, was part of the original software